app/src/main/java/org/kiwix/kiwixmobile/utils/SharedPreferenceUtil.java

```diff
@@ -0,0 +1,51 @@
+package org.kiwix.kiwixmobile.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import javax.inject.Inject;
+
+/**
+ * Created by srv_twry on 25/2/18.
+ * Manager for the Default Shared Preferences of the application.
+ */
+
```

Comment:
As I can see you're using constructor injection, so use `@javax.inject.Singleton` here for annotating the class so that **Dagger** only keeps one object of **SharedPreferenceUtil** in memory.